### ISSUE TITLE
[Accessibility, a11y] Screen reader isn't reading quick pick title field

### ISSUE THREAD

**Issue by MicroFish91** [microsoft/vscode#149387]
**VS Code Version**:  1.67.1

**Operating System**:  Windows 11 / macOS v14.0

**Screen Reader**:  NVDA / Apple VoiceOver

**Extensions Installed**:
Put together an extension to quickly demo the bare VS Code UI features without our normal SDK and extension wrappers.
"a11y-kit" (https://marketplace.visualstudio.com/items?itemName=MicroFish.accessibility-testkit)

**Repro Steps**:
  1. Install a11y-kit from VS Code Marketplace
  2. Open command palette and run a11y-kit's menu command
  3. Next, select: "Quick Picks: Show Quick Picks - Single" and observe.

**Expected**:
When entering the quick pick selection screen, the title should be read by the screen reader so that the user understands the context of the selection environment.

**Actual**:
Certain other descriptors are picked up and read, but not the "Quick Pick Placeholder Title".

Note:  A user posted an issue on one of our extensions explaining that the title is picked up on JAWS but not NVDA.  I have only verified this on VoiceOver and NVDA - neither screen reader reads the title as far as I can tell.

**Attachments**:
![sqp_macos](https://user-images.githubusercontent.com/40250218/168152938-07b1d627-7861-435e-abd5-a05e9d9faf30.gif)

Thank you

**Comment by sbatten** [comment]
Title is still not read

**Comment by isidorn** [comment]
@TylerLeonhardt feel free to push out to next milestone. I leave this up to you :)

**Comment by TylerLeonhardt** [comment]
Yeah I don't know I just tried this again and it's 50-50 whether the ARIA label is read. If it _is_ read, it has the title in it.

**Comment by isidorn** [comment]
@TylerLeonhardt it is up to the screen reader to decide if it is read. If the title is now append when it gets read then we are good.
Let's close and give @MicroFish91 time to verify. I can also bring it up with the screen reader folk if needed.

**Comment by MicroFish91** [comment]
@TylerLeonhardt @isidorn it seems as though the title is only picked up in these two cases (see red arrows) when I test with a11y-kit.  The titles in the other 6 options are not being picked up by NVDA.

![image](https://user-images.githubusercontent.com/40250218/174356681-ec084bb9-d5eb-4a57-b1af-c13d75001bf6.png)

Do you guys think this is something that can be fixed on your end, or is this an issue with the way the screen readers are picking up the label?

Thanks again for your continued help!

**Comment by isidorn** [comment]
@MicroFish91 since this can be reproduced both with VoiceOver and NVDA the issue is most likely Chrome.
So can you open a new issue here https://bugs.chromium.org/p/chromium/issues/list

Make sure to use https://vscode.dev/ for repro steps when you file it to Chromium so we are sure that Electron is not causing this.

**Comment by MicroFish91** [comment]
Is there anyone on the VS Code team that could tackle this one?  I feel that someone on the originating team is likely to provide higher quality context + repro steps and is better suited to answer any follow-up questions that might result.

Thanks again for your time and continued help😊

**Comment by isidorn** [comment]
@MicroFish91 I could do it, but I am time constrained this week, and next 3 weeks I am on vacation. So I could do it in August.

**Comment by MicroFish91** [comment]
No worries, I don't think there's an issue with the August timeframe.  Thanks and enjoy your vacation!

**Comment by isidorn** [comment]
@MicroFish91 your  `a11y-kit` does not work for vscode.dev. Can you please transition it to a web extension https://code.visualstudio.com/api/extension-guides/web-extensions

I think you do not depend on any node api so it might be just that you declare it as a web extension.

**Comment by MicroFish91** [comment]
> @MicroFish91 your `a11y-kit` does not work for vscode.dev. Can you please transition it to a web extension https://code.visualstudio.com/api/extension-guides/web-extensions
> 
> I think you do not depend on any node api so it might be just that you declare it as a web extension.

I believe it should be transitioned to a web extension now, thanks for that heads up!  Let me know if you have any issues running it.

**Comment by isidorn** [comment]
Upstream https://bugs.chromium.org/p/chromium/issues/detail?id=1384537

**Comment by MicroFish91** [comment]
Update: We also have a similar issue with VoiceOver logged against our Functions extension.  I'm consolidating the issues here for reference and editing the main issue to apply to both Windows and Mac.  Let me know if you prefer me to open a new issue.  Thank you!

Reference issue: https://github.com/microsoft/vscode-azurefunctions/issues/3819

**Comment by kupatkar99** [comment]
Verified the issue in environment:
Visual studio Code Version: 1.82.1 (Universal)
Issue is fixed. hence closing the bug please find added attcahment for reference.
<img width="1440" alt="MicrosoftTeams-image (1)" src="https://github.com/microsoft/vscode/assets/98530069/59b67265-d035-4bbc-acc7-2f0eb9149e79">
